### PR TITLE
mahmoud-gamal

### PR DESCRIPTION
**Pull request by Cofa12**

### CODE REVIEW

**Review by imdhemy — CHANGES_REQUESTED** [review]
Could you please follow the instructions here

https://github.com/imdhemy/d-mentor/blob/main/request-instructions.md

File: participants/1-introduction/Cofa12/mahmoud gamal [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# mahmoud gamal
```
Comment (imdhemy) [inline participants/1-introduction/Cofa12/mahmoud gamal]: Could you please delete this file? It's not required, we already have the `Readme.md` file for that :)

**Review by imdhemy — CHANGES_REQUESTED** [review]
Could you please address my comment? @Cofa12

File: participants/1-introduction/Cofa12/mahmoud gamal (hunk 1 above)
Comment (Cofa12) [reply participants/1-introduction/Cofa12/mahmoud gamal]: I have done it

**Pull request closed without merging**